dolphinscheduler-task-plugin/dolphinscheduler-task-openmldb/src/main/java/org/apache/dolphinscheduler/plugin/task/openmldb/OpenmldbTask.java

```diff
@@ -0,0 +1,259 @@
+        }
+    }
+
+    @Override
+    public void handle() throws Exception {
+        try {
+            // generate the content of this python script
+            String pythonScriptContent = buildPythonScriptContent();
+            // generate the file path of this python script
+            String pythonScriptFile = buildPythonCommandFilePath();
+
+            // create this file
+            createPythonCommandFileIfNotExists(pythonScriptContent, pythonScriptFile);
+            String command = buildPythonExecuteCommand(pythonScriptFile);
+
+            TaskResponse taskResponse = shellCommandExecutor.run(command);
+            setExitStatusCode(taskResponse.getExitStatusCode());
+            setAppIds(taskResponse.getAppIds());
+            setProcessId(taskResponse.getProcessId());
+            setVarPool(shellCommandExecutor.getVarPool());
+        } catch (Exception e) {
+            logger.error("openmldb task failure", e);
+            setExitStatusCode(TaskConstants.EXIT_CODE_FAILURE);
+            throw new TaskException("run openmldb task error", e);
+        }
+    }
+
+    @Override
+    public void cancelApplication(boolean cancelApplication) throws Exception {
+        // cancel process
+        shellCommandExecutor.cancelApplication();
+    }
+
+    @Override
+    public AbstractParameters getParameters() {
+        return openmldbParameters;
+    }
+
+    /**
+     * create python command file if not exists
+     *
+     * @param pythonScript     exec python script
+     * @param pythonScriptFile python script file
+     * @throws IOException io exception
+     */
+    protected void createPythonCommandFileIfNotExists(String pythonScript, String pythonScriptFile) throws IOException {
+        logger.info("tenantCode :{}, task dir:{}", taskRequest.getTenantCode(), taskRequest.getExecutePath());
+
+        if (!Files.exists(Paths.get(pythonScriptFile))) {
+            logger.info("generate python script file:{}", pythonScriptFile);
+
+            StringBuilder sb = new StringBuilder();
+            sb.append("#-*- encoding=utf8 -*-\n");
+
+            sb.append("\n\n");
+            sb.append(pythonScript);
+            logger.info(sb.toString());
+
+            // write data to file
+            FileUtils.writeStringToFile(new File(pythonScriptFile),
+                    sb.toString(),
+                    StandardCharsets.UTF_8);
+        }
+    }
+
+    /**
+     * build python command file path
+     *
+     * @return python command file path
+     */
+    protected String buildPythonCommandFilePath() {
+        return String.format("%s/openmldb_%s.py", taskRequest.getExecutePath(), taskRequest.getTaskAppId());
+    }
+
+    /**
+     * build python script content
+     *
+     * @return raw python script
+     */
+    private String buildPythonScriptContent() {
+        // sqls doesn't need \n, use ; to split
+        String rawSqlScript = openmldbParameters.getSql().replaceAll("[\\r]?\\n", " ");
```

Comment:
Hi, I don't think you can simply replace all line breaks with spaces, if there are comments in the sql scripts, you will totally change the logic of it, for example, if the original sql is 

```sql
select * from users
-- some comment
inner join order on users.order_id = order.id;
```

will become 

```sql
select * from users -- some comment inner join order on users.order_id = order.id;
```

Reply:
Yes, I should replace \r\n to '/\n' in string, not the space. I'll fix it and add ut